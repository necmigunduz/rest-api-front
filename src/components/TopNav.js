const Nav = () => (
  <div className="background-dark-blue fw-900 m-b-30 nav fixed">
    <h1 className="color-white title bold">BodyMonitor</h1>
  </div>
);

export default Nav;
