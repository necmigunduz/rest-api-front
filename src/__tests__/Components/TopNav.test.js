import renderer from 'react-test-renderer';
import TopNav from '../../components/TopNav';

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useLocation: () => ({
    pathname: 'www.instance.com/test',
  }),
}));

describe('TopNav', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<TopNav />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
