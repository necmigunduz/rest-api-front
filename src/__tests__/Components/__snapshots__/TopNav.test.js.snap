// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNav renders correctly 1`] = `
<div
  className="background-dark-blue fw-900 m-b-30 nav fixed"
>
  <h1
    className="color-white title bold"
  >
    BodyMonitor
  </h1>
</div>
`;
